data/json/npcs/talk_tags.json

```
@@ -489,7 +564,11 @@
       "I saw that! Drop what you just stole!",
       "Thieves will not last long around me <name_g>, please drop that.",
       "Consider this a warning <name_b>, thieves will not be tolerated, drop it.",
-      "You think I'm <swear> blind <name_b>? Don't touch our stuff."
+      "You think I'm <swear> blind <name_b>? Don't touch our stuff.",
+      "Stop right there, criminal scum! You haved violated the law. Forfeit your ill-gotten gains or face punishment.",
```

```suggestion
      "Stop right there, criminal scum! You have violated the law. Forfeit your ill-gotten gains or face punishment.",
```
Too straightforward of a reference IMO. Should be dropped.